### ISSUE TITLE
Provide instructions to install valid cert

### ISSUE THREAD

**Issue by ChaosExAnima** [10up/wp-local-docker-v2#22]
I think it would be useful to provide guidance as to how to set up a valid certificate for a given domain. There's definitely situations where external JS scripts don't work properly if it doesn't detect a valid cert, and currently there's no easy way around that.

**Comment by joseffb-mla** [comment]
Anyone figure this out yet or point to a blog post on how to install in a 10up docker image?

**Comment by joseffb-mla** [comment]
Figured it out:

create your certificate. I use this script to make local wild card certs:

```
openssl req \
    -newkey rsa:2048 \
    -x509 \
    -nodes \
    -keyout ~/Desktop/mysite.test.key \
    -new \
    -out ~/Desktop/mysite.test.crt \
    -reqexts SAN \
    -extensions SAN \
    -config <(cat /System/Library/OpenSSL/openssl.cnf \
        <(printf '
[req]
default_bits = 2048
prompt = no
default_md = sha256
x509_extensions = v3_req
distinguished_name = dn

[dn]
C = JP
ST = Tokyo
L = Tokyo
O = MyDomain Inc.
OU = Technology Group
emailAddress = hello@mysite.test
CN = mysite.test

[v3_req]
subjectAltName = @alt_names

[SAN]
subjectAltName = @alt_names

[alt_names]
DNS.1 = *.mysite.test
DNS.2 = mysite.test
')) \
    -sha256 \
    -days 3650
```

copy that key and crt file to 
$ /Users/username/wp-local-docker-sites/mysite-test/config/nginx/certs 
$ /Users/username/Documents/Kitematic/global_gateway_1/etc/nginx/certs 

edit the docker-compose.yml:
  add to nginx->volume: 
    - './config/nginx/certs:/etc/nginx/certs:cached'
  edit nginx->environment->CERT_NAME: 
    from localhost to the name of your crt file (no .crt i.e. mysite.crt put mysite)

stop all, clear cache, start. 
edit

**Comment by eugene-manuilov** [comment]
The next release `2.8.0` will include commands to install local CA in the system trust store and to generate certificates for local environments. We will also update the documentation to explain how to use it.

**Comment by eugene-manuilov** [comment]
It has been implemented and merged via #138. We can consider this ticket as done. (Currently can be tested by installing `@next` version).
